tasks/broad/JointGenotypingTasks.wdl

```diff
@@ -999,13 +999,19 @@ task GetFingerprintingIntervalIndices {
 
     cp ~{sep=' ' unpadded_intervals} ./
 
-    cat ~{write_lines(unpadded_intervals)} | xargs -n1 basename | xargs -I{} bash -c 'rename_scatter $@' _ {}
+    while read line; do
+      name=$(basename $line)
+      rename_scatter $name
+    done < ~{write_lines(unpadded_intervals)}
 
     #find the first header
-    find . -name "scattered.renamed.*.interval_list" | head -n1 | xargs cat | grep '^@' > all.interval_list
+    first_header_file=$(find . -name 'scattered.renamed.*.interval_list' -print -quit)
```

Comment:
We could benefit from some local variables here.  I'm a little anxious about hard coding "scattered.renamed" in two places.  If you're going to make that change you might want to add "@" as "interval_list_comment_char" or something.